protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -501,8 +504,25 @@ func (k Keeper) internalCanUpdateSubaccounts(
 			)
 		}
 
-		if exists && currentBlock-lastBlockNegativeTncSubaccountSeen <
-			types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS {
+		// Panic if the current block is less than the last block a chain outage was seen.
+		downtimeInfo := k.blocktimeKeeper.GetDowntimeInfoFor(ctx, 5*time.Minute)
+		if currentBlock < downtimeInfo.BlockInfo.Height {
+			panic(
+				fmt.Sprintf(
+					"internalCanUpdateSubaccounts: current block (%d) is less than the last "+
+						"block a chain outage was seen (%d)",
+					currentBlock,
+					downtimeInfo.BlockInfo.Height,
+				),
+			)
+		}
+
+		negativeTncSubaccountSeen := exists && currentBlock-lastBlockNegativeTncSubaccountSeen <
+			types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS
+		chainOutageSeen := downtimeInfo.BlockInfo.Height > 0 && currentBlock-downtimeInfo.BlockInfo.Height <
```

Comment:
I was initially confused by this line. Might help to add clarity here for those who are new to the downtime info struct stored in state.

I'm also worried about a bug accidentally being introduced here later. Right now this will set the bool to true if there is any chain halt with duration <= 5 minutes. Shouldn't we be checking for exactly 5 minutes? If another duration (say 1 min) is introduced for some other use case then this bool would now be true if the outage was just 1 min long